app/src/main/res/drawable/ic_volume_up_vector.xml

```diff
@@ -0,0 +1,3 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android" android:width="24dp" android:height="24dp" android:viewportWidth="24" android:viewportHeight="24">
+    <path android:fillColor="#FFFFFF" android:pathData="M3 10v4c0 0.55 0.45 1 1 1h3l3.29 3.29c0.63 0.63 1.71 0.18 1.71-0.71V6.41c0-0.89-1.08-1.34-1.71-0.71L7 9H4c-0.55 0-1 0.45-1 1zm13.5 2c0-1.77-1.02-3.29-2.5-4.03v8.05c1.48-0.73 2.5-2.25 2.5-4.02zM14 4.45v0.2c0 0.38 0.25 0.71 0.6 0.85C17.18 6.53 19 9.06 19 12s-1.82 5.47-4.4 6.5c-0.36 0.14-0.6 0.47-0.6 0.85v0.2c0 0.63 0.63 1.07 1.21 0.85C18.6 19.11 21 15.84 21 12s-2.4-7.11-5.79-8.4C14.63 3.37 14 3.82 14 4.45z"/>
```

Comment:
lets be consistent here, always use @android:color/white or #FFFFFFFF

Reply:
Should we also do this in commons? Replacing all usages with "@android:color/white"

Reply:
I was using #FFFFFFFF mostly, no need to replace it now. Just specify alpha

Reply:
Oops, I just did that. Reverting...

Reply:
all right :)